### DISCUSSION TITLE
Can encoded URLs be used for the logo?

### DISCUSSION THREAD

**Discussion by Andre601** [badges/shields#7697] [Q&A] [answered]
Was wondering if you - instead of using Base64-encoded SVG data - can use a URL instead that points to an SVG to use.

I just now made a modified version of simpleicons.now.sh, which allows you to have Simpleicons coloured, for Octicons and of course would like to try and use that for badges, if doable.

I can understand if that isn't working and will accept the current approach of Base64-encoded SVG data as the alternative.

**Comment by calebcartwright** [comment, marked as answer]
> I just now made a modified version of simpleicons.now.sh, which allows you to have Simpleicons coloured, for Octicons and of course would like to try and use that for badges, if doable.

Thanks for sharing! Please feel free to follow up if you get something working, even just an enumeration of examples as that seems like something that might warrant inclusion in [Awesome Badges](https://github.com/badges/awesome-badges) (both for those interested in Octicons specifically or even the pattern more generally).

> can use a URL instead that points to an SVG to use.

As to the main question :point_up:, it's a duplicate of several prior questions/requests (e.g. #7404, #6153, #760, #2186, etc.) and not something we're going to support, regardless of whether the url would be encoded or not.

**Reply by Andre601** [reply]
> Thanks for sharing! Please feel free to follow up if you get something working, even just an enumeration of examples as that seems like something that might warrant inclusion in [Awesome Badges](https://github.com/badges/awesome-badges) (both for those interested in Octicons specifically or even the pattern more generally).

Well, the site I have is based off another one I mentioned in another discussion/issue - namely https://simpleicons.vercel.app - and does pretty much the same but for Octicons of course.

Perhaps the Awesome badges Readme could include a section about sites and/or tools for SVG customization or smth... I leave this up to discuss.

> As to the main question ☝️, it's a duplicate of several prior questions/requests (e.g. #7404, #6153, #760, #2186, etc.) and not something we're going to support, regardless of whether the URL would be encoded or not.

Are there any particular reasons (performance, security, etc.) for why? Curious to know the details behind this decision.
